recipes/tinyalign/meta.yaml

```diff
@@ -5,7 +5,7 @@ package:
   version: {{ version }}
 
 source:
-  url: https://files.pythonhosted.org/packages/20/2c/a5c5f356e3d491941fe29aba4d71078e90eb4ea83b26af616b1ac9502ab7/tinyalign-0.2.tar.gz
+  url: https://pypi.io/packages/source/t/tinyalign/tinyalign-{{ version }}.tar.gz
```

Comment:
This is just so we can be a little more lazy and only update `version` and `sha256` on updates.

Reply:
Ah, perfect! I didn’t know these short URLs existed and was always annoyed that `{{ version }}` couldn’t be used with the long ones.